app/javascript/gobierto_data/webapp/components/sets/SQLEditor.vue

```diff
@@ -2,6 +2,7 @@
   <div>
     <div class="gobierto-data-sql-editor">
       <SQLEditorHeader
+        v-if="publicQueries"
```

Comment:
could `publicQueries` come as `undefined` or `null`? Because if it's an array, you should check `v-if="publicQueries.instead"`